qemu/tests/readonly_disk.py

```diff
@@ -60,16 +76,15 @@ def run(test, params, env):
     error_context.context(
         "TEST STEPS 4: Write to the readonly disk expect:"
         "The media is write protected", logging.info)
-    dst_file_readonly = params["dst_file_readonly"]
+    dst_file_readonly = params["dst_file_readonly"] % drive_letter[0]
```

Comment:
According to 52 line,  if disks have 2 or more values, here  drive_letter[0] only get the value for last time assignment, it should be assigned value for each disk~